### ISSUE TITLE
Setting the Tab Size

### ISSUE THREAD

**Issue by CaptainPackers** [MarcosBL/dokuwiki-plugin-autotabber#5]
I don't understand how to reset the tab size. There are three JS files. Could you specify exactly which file needs to be modified and how? I tried changing

`tabOverride.set(textareas).tabSize(0).autoIndent(true).escape(true);`
to
`tabOverride.set(textareas).tabSize(2).autoIndent(true).escape(true);`

in script.js but it had no affect. I also tried changing the line to

`tabOverride.set(textareas).2.autoIndent(true).escape(true);` 

which also had no affect.

I really like the concept of your plugin, but I need to modify the tab to two spaces to be able to use it.

Thanks

**Comment by MarcosBL** [comment]
I no longer use Dokuwiki anymore, but the file/line to be edited should be https://github.com/MarcosBL/dokuwiki-plugin-autotabber/blob/master/script.js#L12

so `tabOverride.set(textareas).tabSize(2).autoIndent(true).escape(true);` should do it, are you sure you are hard-reloading the page after the change ? no cache, etc ?

**Comment by CaptainPackers** [comment]
Thanks.

Does this mean you won't be supporting the plugin anymore?

Regards,
Donald Lund

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by MarcosBL** [comment]
No, that's not a problem, the plugin is very small, and easily maintenable; I'll happily accept and merge any PR, and even take a look at it myself, is just as I no longer have an updated installation, it could take a bit of time

**Comment by CaptainPackers** [comment]
That's good to hear. Thanks. Appreciate the plugin.

Regards,
Donald Lund

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
